agent/setup.go

```diff
@@ -80,6 +80,9 @@ func NewBaseDeps(configLoader ConfigLoader, logOut io.Writer) (BaseDeps, error)
 
 	d.RuntimeConfig = cfg
 	d.Tokens = new(token.Store)
+	// TODO: why do we ignore failure to load persisted tokens?
+	_ = d.Tokens.Load(cfg.ACLTokens, d.Logger)
```

Comment:
We previously ignored this error, but I'm not sure why we ignore it on the first load.